prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck{
+      display: flex;
+      flex-basis: calc(100%/3);
```

Comment:
<img width="1728" alt="Screen Shot 2022-02-28 at 11 32 13 PM" src="https://user-images.githubusercontent.com/99834589/156124742-8ff2f4ff-12c6-497d-a733-2873ace8b3f3.png">
<img width="1728" alt="Screen Shot 2022-02-28 at 11 32 10 PM" src="https://user-images.githubusercontent.com/99834589/156124754-8cec0e97-316e-43b9-ae26-ed61ee64ae79.png">
<img width="1728" alt="Screen Shot 2022-02-28 at 11 32 07 PM" src="https://user-images.githubusercontent.com/99834589/156124767-d12db1dd-21eb-49bb-a9cf-f530744d92fc.png">

Flex-basis property applied to .card-wrapper instead of #card-deck for 3 of the HTML documents that had flex-basis properties